config/locales/en.yml

```diff
@@ -32,6 +32,11 @@ en:
     private_topics:
       errors:
         user_ids_length: Please specify at least one other user.
+    search:
+      form:
+        btn_submit: :thredded.search.form.label
```

Comment:
Full disclosure - I'm an i18n dummy. I haven't fully i18n'd an app to this day (I regret that, by the way).

That being said - is it common practice to shorten "button" to "btn"? If that's not a convention, I'd lean towards expanding this out. If it's a convention, please ignore me :)